packages/generator/src/edmx-to-vdm/common/operation-parameter.ts

```diff
@@ -14,38 +14,11 @@ import { getTypeMappingActionFunction } from '../edmx-to-vdm-util';
 /**
  * @internal
  */
-export function getActionImportParameters(
-  edmxActionImport: EdmxActionImport,
-  edmxParameters: EdmxParameter[],
-  swaggerDefinition: SwaggerPath | undefined,
-  formatter: ServiceNameFormatter
-): VdmParameter[] {
-  return getParameter(
-    edmxActionImport,
-    edmxParameters,
-    swaggerDefinition,
-    formatter
-  );
-}
-/**
- * @internal
- */
-export function getFunctionImportParameters(
-  edmxFunctionImport: EdmxFunctionImportV2 | EdmxFunctionImportV4,
-  edmxParameters: EdmxParameter[],
-  swaggerDefinition: SwaggerPath | undefined,
-  formatter: ServiceNameFormatter
-): VdmParameter[] {
-  return getParameter(
-    edmxFunctionImport,
-    edmxParameters,
-    swaggerDefinition,
-    formatter
-  );
-}
-
-function getParameter<T extends EdmxNamed>(
-  edmxActionFunctionImport: T,
+export function getOperationParameters(
+  edmxActionFunctionImport:
+    | EdmxFunctionImportV2
```

Comment:
[req] I guess you can use the `EdmxOperation` here as well?

Reply:
I guess you mean `EdmxOperationImport`? If so, done.